### ISSUE TITLE
Feature Request - Groups

### ISSUE THREAD

**Issue by ghost** [Gaarnik/DimStorage#14]
Currently we have Public and Private option. It would help a ton for server play to have Group option. Where you can share with a group of other players. While making this it would also nice to have the option to have more then one group that can have more then one owner. 

Example: Where you can have two or more people that can add/remove other people a group.

**Comment by Phenix246** [comment]
We can try to add a friends system, it could be a good think for the mod, I go to work on

**Comment by Gaarnik** [comment]
Yes, it could be a good feature, but for now I have other plan for future updates. Maybe later !
